### DISCUSSION TITLE
OpenAI Swarm using Ell

### DISCUSSION THREAD

**Discussion by gurvinder-dhillon** [MadcowD/ell#322] [Show and tell]
Everyone is talking about openai swarm which is experimental and educational.
We need people to know about ell.
I have been creating those agent swarms using ell and same concept of passing tools as agents for sometime now, even before openai published swarm.

Swarm is getting so much attention, we need to let people know how easy it is to do the same thing with ell which is much more mature and user friendly and not an experimental project.

I have put an example of replicating the example written by openai
Ell Swarm - Triage Agent : https://github.com/gurvinder-dhillon/playground/blob/main/playground/ell_swarm.ipynb
OpenAi Swarm - Triage Agent : https://github.com/openai/swarm/blob/main/examples/triage_agent/agents.py

And since people love attention, I recorded a video as well - https://www.youtube.com/watch?v=kVgMzj-pEiI

**Comment by SamuelDevdas** [comment]
Please reshare your jupyter link, unfortunately it does not work for me, maybe it is set to private?
